packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/data_sources/local/auth_data_source.dart

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2021, Prime Holding JSC
+// https://www.primeholding.com
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
+
+class AuthDataSource {
```

Comment:
The name `AuthDataSource` may be considered ambiguous as it is not only referring to an auth token but auth in general. Maybe change it to something like `AuthTokenDataSource`.